test/cpp/interop/metrics_client.cc

```diff
@@ -27,6 +27,7 @@
 #include "src/core/lib/gprpp/crash.h"
 #include "src/proto/grpc/testing/metrics.grpc.pb.h"
 #include "src/proto/grpc/testing/metrics.pb.h"
+#include "test/core/test_util/test_config.h"
 #include "test/cpp/util/metrics_server.h"
 #include "test/cpp/util/test_config.h"
 
```

Comment:
I hope it is ok to put a "test/core/test_util" header into ""test/cpp/util" folder.